src/WiFi101.h

```diff
@@ -61,6 +61,12 @@ typedef enum {
 	WL_AP_MODE
 } wl_mode_t;
 
+#define WL_PING_SUCCESS 0
+#define WL_PING_DEST_UNREACHABLE 1
+#define WL_PING_TIMEOUT 2
+#define WL_PING_UNKNOWN_HOST 3
+#define WL_PING_ERROR 4	
+
```

Comment:
@sandeepmistry sure, I will change this to enum. ping_cb will need to be refined to reflect changes.
We will assume u8ErrorCode in tpfPingCb will have only three values [PING_ERR_SUCCESS, PING_ERR_DEST_UNREACH, PING_ERR_TIMEOUT](http://asf.atmel.com/docs/latest/samd21/html/group___async_calback.html#ga727217f41dd75101c33bc35dced1de30)